cmd/kpod/spec.go

```diff
@@ -17,6 +17,31 @@ import (
 	"golang.org/x/sys/unix"
 )
 
+func addPidNS(config *createConfig, g *generate.Generator) error {
```

Comment:
This looks like it's from #54 except it seems to be missing the latest changes from there

Reply:
That's why this is a WIP, waiting for it to get merged, then I can rebase and remove WIP